src/config_generator/snapshots/tailcall__config_generator__from_proto__test__from_proto.snap

```diff
@@ -0,0 +1,65 @@
+---
+source: src/config_generator/from_proto.rs
+expression: result
+---
+schema @server @upstream {
+  query: Queryx
+}
+
+input GREETINGS_DUP_HelloRequest @tag(name: "greetings_dup.HelloRequest")  {
+  name: String
+}
+
+input GREETINGS_HelloRequest @tag(name: "greetings.HelloRequest")  {
+  name: String
+}
+
+input NEWS_Author @tag(name: "news.Author")  {
+  email: String
+  name: String
+}
+
+input NEWS_MultipleNewsId @tag(name: "news.MultipleNewsId")  {
+  ids: [NEWS_NewsId]
+}
+
+input NEWS_News @tag(name: "news.News")  {
```

Comment:
```suggestion
input NewsNews @tag(name: "news.News")  {
```

Slight change in the original request. Let's use camel case only for the names. No snake casing at all.